releases/net471/KnownIssues/510901-Apps-targeting-.NET-4.6.1-who-using-.NET-Standard-libraries-might-be-broken-on-.NET-4.7.1.md

```diff
@@ -0,0 +1,18 @@
+## Symptoms
+
+Application that target .NET Framework 4.6.1 who use .NET Standard libraries could start seeing TypeLoadExceptions when running on .NET 4.7.1
+
+## Cause
+
+.NET Framework 4.7.1 adds support for .NET Standard in-box. As part of this change, a few assemblies who used to ship independently from the .NET Framework have now been added to the .NET Framework.
+
+Because those assemblies were not part of the .NET Framework prior to .NET 4.7.1 they were deployed with the application and a binding redirect was added to your application's app.config.
```

Comment:
No paragraph break here -- this should be included with the previous paragraph.
.NET 4.7.1 --> the .NET Framework 4.7.1
application and --> application, and 
your application's app.config --> the application's app.config file.